thunder/benchmarks/benchmark_peft.py

```diff
@@ -0,0 +1,724 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import random
+import time
+from contextlib import contextmanager
+from distutils.version import LooseVersion
+from typing import Any, List, Optional
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+import transformers
+from datasets import Dataset
+from loguru import logger
+from peft import LoraConfig, get_peft_model
+from torch.distributed import DeviceMesh, init_process_group
+from torch.distributed._composable.fsdp import fully_shard
+from torch.nn.attention import SDPBackend, sdpa_kernel
+from tqdm import tqdm
+from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer
```

Comment:
Could you clean up these imports? At glance, I'm not quite convinced with the imports of numpy and transformers.
For transformers import, we do import three from it so I think it'd be a bit cleaner to avoid `import transformers`